contest/contest_info.py

```diff
@@ -63,10 +63,38 @@ def get_running_contests():
 def get_contestant_list(contest):
     return Contestant.objects.filter(contest = contest)
 
+
+def get_contestant(contest):
+    contestants = Contestant.objects.filter(contest=contest)
+    return [contestant.user for contestant in contestants]
```

Comment:
Any possible to get users in DB level?
It seems use db obj will be more elegant and efficient?